src/app/shell/admin-tools/data/directions-wrapper/directions/create-direction/create-direction.component.ts

```diff
@@ -1,22 +1,22 @@
-import { CreateDirection, GetDirectionById, UpdateDirection } from './../../../../../../shared/store/admin.actions';
-import { takeUntil, filter } from 'rxjs/operators';
 import { STEPPER_GLOBAL_OPTIONS } from '@angular/cdk/stepper';
 import { Component, OnDestroy, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatDialog } from '@angular/material/dialog';
+import { ActivatedRoute, Router } from '@angular/router';
 import { Select, Store } from '@ngxs/store';
 import { Observable } from 'rxjs';
-import { Location } from '@angular/common';
-import { FormGroup, FormBuilder, FormControl, Validators } from '@angular/forms';
-import { MatDialog } from '@angular/material/dialog';
-import { ConfirmationModalWindowComponent } from '../../../../../../shared/components/confirmation-modal-window/confirmation-modal-window.component';
-import { Constants } from '../../../../../../shared/constants/constants';
-import { ModalConfirmationType } from '../../../../../../shared/enum/modal-confirmation';
-import { NavBarName } from '../../../../../../shared/enum/enumUA/navigation-bar';
-import { Direction } from '../../../../../../shared/models/category.model';
-import { NavigationBarService } from '../../../../../../shared/services/navigation-bar/navigation-bar.service';
-import { AdminState } from '../../../../../../shared/store/admin.state';
-import { AddNavPath } from '../../../../../../shared/store/navigation.actions';
+import { filter, takeUntil } from 'rxjs/operators';
+
+import { ConfirmationModalWindowComponent } from 'shared/components/confirmation-modal-window/confirmation-modal-window.component';
+import { Constants } from 'shared/constants/constants';
+import { NavBarName } from 'shared/enum/enumUA/navigation-bar';
+import { ModalConfirmationType } from 'shared/enum/modal-confirmation';
+import { Direction } from 'shared/models/category.model';
+import { NavigationBarService } from 'shared/services/navigation-bar/navigation-bar.service';
+import { AdminState } from 'shared/store/admin.state';
+import { AddNavPath } from 'shared/store/navigation.actions';
```

Comment:
It is OK for replacing such long relative path to shared folder for directly using shared because of paths that specified in tsconfig.json?

Reply:
It's ok, it just makes importing easier